android/app/build.gradle

```diff
@@ -113,4 +113,8 @@ dependencies {
     releaseCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.3.1' // or 1.4-beta1
     testCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.3.1' // or 1.4-beta1
 
+    // For UI improvements
+    compile 'me.zhanghai.android.materialprogressbar:library:1.1.7'
+    compile 'com.balysv:material-ripple:1.0.2'
```

Comment:
Okay. First library was unnecessary. I removed it. Although ripples are better created by using that library, custom is problematic.